sass/main.scss

```diff
@@ -343,6 +343,59 @@ nav {
         }
 
     }
+    #modal {
+        display: none;
+        position: fixed;
+        z-index: 100;
+        padding-top: 27px;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 100%;
+        overflow: auto;
+        background-color:rgba(0,0,0) ;
+        background-color: rgba(0,0,0,0.9);
+    }
+
+    #modal-content {
+        margin: auto;
+        display: block;
+        width: 80%;
+        max-width: 700px;
+        -webkit-animation-name: zoom;
+        -webkit-animation-duration: 0.6s;
+        animation-name: zoom;
+        animation-duration: 0.6s;
+    }
+
+    @-webkit-keyframes zoom {
+        from {-webkit-transform: scale(0)}
+        to {transform: scale(1)}
+    }
+
+    @keyframes zoom {
+        from {transform:scale(0)}
+        to {transform: scale(1);}
+    }
+
+    #close {
+        position: absolute;
+        top: 15px;
+        right: 23px;
```

Comment:
Imo it's cleaner to just do
```css
top: 20px
right: 20px
```
But see my comments in the overall review for moving this